src/main/anovos/data_analyzer/stats_generator.py

```diff
@@ -493,7 +493,14 @@ def measures_of_centralTendency(
         summary_lst.append(summary_col)
```

Comment:
In here, what I would recommend is, before appending to `summary_lst`, make sure `mean`, `50%` and `count` follows the same datatype as you mentioned below. Thus, we will need to `cast` the data before appending to `summary_lst`.